src/components/Drawer/mdc-temporary-drawer.vue

```diff
@@ -0,0 +1,114 @@
+<!-- thanks to petejohanson for this implementation -->
+
+<template>
+  <aside class="mdc-temporary-drawer mdc-typography" :class="classes">
+    <nav ref="drawer" class="mdc-temporary-drawer__drawer">
+
+      <div class="mdc-temporary-drawer__toolbar-spacer" v-if="$slots['toolbar-spacer'] || toolbarSpacer">
+        <slot name="toolbar-spacer"/>
```

Comment:
Yes I think it will definitely depend on the component. Also, I think some of that implementation will be separated into the thin layer and the sugar components at a later time. So we aren't forcing a dev to use slots.